tools/nni_cmd/config_utils.py

```diff
@@ -81,6 +81,8 @@ def add_experiment(self, expId, port, time, file_name, platform):
         self.experiments[expId]['status'] = 'INITIALIZED'
         self.experiments[expId]['fileName'] = file_name
         self.experiments[expId]['platform'] = platform
+        self.experiments[expId]['authorName'] = author_name
```

Comment:
what is `author_name` used for?

Reply:
It's an important metadata. Save it for future use. Also okay to eject. Suggestions?

Reply:
Don't know if it will be used in the future, do not suggest to maintain unuseful information in metadata.